node.gyp

```diff
@@ -920,6 +920,143 @@
+              'obj_suffix': 'obj',
+            }],
+            ['GENERATOR=="ninja"', {
+              'obj_path': '<(obj_dir)/src',
+              'obj_gen_path': '<(obj_dir)/gen',
+              'obj_insp_path': '<(obj_dir)/src/inspector',
+              'obj_tracing_path': '<(obj_dir)/src/tracing',
+              'obj_separator': '/<(node_lib_target_name).',
+            }, {
+              'conditions': [
+                ['OS=="win"', {
+                  'obj_path': '<(obj_dir)/<(node_lib_target_name)',
+                  'obj_gen_path': '<(obj_dir)/<(node_lib_target_name)',
+                  'obj_insp_path': '<(obj_dir)/<(node_lib_target_name)',
+                  'obj_tracing_path': '<(obj_dir)/<(node_lib_target_name)',
+                }],
+              ]}
+            ]
+          ],
+        },
+        'var_defines': [],
+        'dep_objs': [
+          '<(obj_path)<(obj_separator)async_wrap.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)bootstrapper.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)callback_scope.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)exceptions.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)fs_event_wrap.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)handle_wrap.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)env.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_api.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_buffer.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_debug_options.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_encoding.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_i18n.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_messaging.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_perf.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_platform.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_url.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_worker.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)util.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)sharedarraybuffer_metadata.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)string_bytes.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)string_decoder.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)stream_base.<(obj_suffix)',
+          '<(obj_path)<(obj_separator)node_constants.<(obj_suffix)',
+          '<(obj_tracing_path)<(obj_separator)agent.<(obj_suffix)',
+          '<(obj_tracing_path)<(obj_separator)node_trace_buffer.<(obj_suffix)',
+          '<(obj_tracing_path)<(obj_separator)node_trace_writer.<(obj_suffix)',
+          '<(obj_tracing_path)<(obj_separator)trace_event.<(obj_suffix)',
+          '<(obj_gen_path)<(obj_separator)node_javascript.<(obj_suffix)',
+        ],
+        'conditions': [
+          [ 'node_use_openssl=="true"', {
+            'var_defines+': [ 'HAVE_OPENSSL=1' ],
```

Comment:
So GYP has default list appending semantics so no need for the `+` suffix
[`+` just means "prepend"](https://gyp.gsrc.io/docs/InputFormatReference.md#Merge-Basics)

Reply:
I ran into some issues when I didn't use `+` sign in other place. In this case, I can remove it.